CONTRIBUTING.md

```diff
@@ -16,9 +16,6 @@ Issues
   keeping the issues properly organized and searchable (by OS, issue type, etc.).
 * When reporting a malfunction, consider enabling
   [debug mode](https://psutil.readthedocs.io/en/latest/#debug-mode) first.
-* To report a **security vulnerability**, use the
-  [Tidelift security contact](https://tidelift.com/security).
-  Tidelift will coordinate the fix and the disclosure of the reported problem.
```

Comment:
I don't think it's harmful to have this info here _as well_. Or, at the very least the policy should be mentioned and linked to from the README. (IMHO.)